product_docs/docs/epas/15/reference/sql_reference/02_data_types/06_xml_type.mdx

```diff
@@ -5,11 +5,20 @@ legacyRedirectsGenerated:
   - "/edb-docs/d/edb-postgres-advanced-server/reference/database-compatibility-for-oracle-developers-reference-guide/9.6/Database_Compatibility_for_Oracle_Developers_Reference_Guide.1.016.html"
   - "/edb-docs/d/edb-postgres-advanced-server/user-guides/database-compatibility-for-oracle-developers-guide/9.5/Database_Compatibility_for_Oracle_Developers_Guide.1.048.html"
 redirects:
-  - /epas/latest/epas_compat_reference/02_the_sql_language/02_data_types/06_xml_type/ #generated for docs/epas/reorg-role-use-case-mode
+  - ../../../../../epas_compat_reference/02_the_sql_language/02_data_types/06_xml_type #generated for docs/epas/reorg-role-use-case-mode
+source:
+   url: https://www.postgresql.org/docs/current/datatype-xml.html
+   timestamp: 2023-09-20
 ---
 
 <div id="xml_type" class="registered_link"></div>
 
+| Name           | Native | Alias | Description                                   |
+| -------------- | ------ | ----- | --------------------------------------------- |
+| `XMLTYPE`      | ✅     |       | Data type used to store XML data.             |
+
+## Overview
+
 The `XMLTYPE` data type is used to store XML data. Its advantage over storing XML data in a character field is that it checks the input values for how well they're formed. Also, support functions perform type-safe operations on it.
```

Comment:
We do not need to include this line and I believe it would be misleading if we did; EPAS ships with XML support, and customers cannot generally rebuild EPAS themselves.